src/main/java/org/javarosa/core/io/StreamsUtil.java

```diff
@@ -95,11 +95,24 @@ public OutputIOException(IOException internal) {
      */
     public static void writeFromInputToOutputUnmanaged(InputStream is,
                                                        OutputStream os) throws IOException {
```

Comment:
Can you split this into "throws InputIOExcepton, OutputIOException"? It's helpful for us to signal that